app/sessionStore.js

```diff
@@ -626,7 +628,8 @@ module.exports.defaultAppState = () => {
       pendingRecords: {}
     },
     locationSiteKeysCache: undefined,
-    sites: getTopSiteMap(),
+    sites: getTopSiteMap(), // TODO remove
```

Comment:
I found out that we only use top sites in new tab, so top sites will be automatically added to the new tab top sites list inside the `newTab` component. Top sites will not be part of the state anymore.